services/galley/schema/src/V37.hs

```diff
@@ -0,0 +1,16 @@
+module V37 (migration) where
+
+import Imports
+import Cassandra.Schema
+import Text.RawString.QQ
+
+migration :: Migration
+migration = Migration 37 "Create table `custom-instances`" $ do
+    schema' [r|
+        CREATE TABLE custom_instances (
+            domain             text,
+            config_json_url    blob,
```

Comment:
why blob, not text?  text is arbitrarily long and utf8, which should be enough for urls.  blob is less restrictive, so reading it back may go wronger.

i have to admit:

```
$ git grep -Hn ur[li] | grep schema/src
services/brig/schema/src/V35.hs:15:            , url      blob
services/brig/schema/src/V35.hs:43:            , base_url     blob
services/galley/schema/src/V20.hs:62:            base_url     blob,
services/galley/schema/src/V31.hs:12:            base_url     blob,
[...]
services/spar/schema/src/V0.hs:41:            , request_uri   text
```

the only other url column that has *not* blob type is one that i wrote.  so i guess the question is: why am i wrong?